README.md

```diff
@@ -30,24 +30,35 @@ Complete configuration for the different [node.js versions](https://github.com/n
 # https://github.com/nodejs/LTS
 sudo: false
 language: node_js
+os:
+ - linux
+before_install:
+  - 'case "${TRAVIS_NODE_VERSION}" in 0.*) export NPM_CONFIG_STRICT_SSL=false ;; esac'
+  - 'nvm install-latest-npm'
+install:
+  - 'if [ "${TRAVIS_NODE_VERSION}" = "0.6" ] || [ "${TRAVIS_NODE_VERSION}" = "0.9" ]; then nvm install --latest-npm 0.8 && npm install && nvm use "${TRAVIS_NODE_VERSION}"; else npm install; fi;'
```

Comment:
Could you explain what each segment in the `before_install` and `install` does, as my travis/bash cripts are nowhere near as gooda as yours it seems?

Reply:
this is because the latest npm version that 0.6 and 0.9 support, doesn't support scoped modules, so it uses node 0.8 to install them and then switches back.